djangocms_form_builder/cms_plugins/legacy.py

```diff
@@ -2,7 +2,7 @@
 from django.conf import settings
 
 if 'djangocms_frontend.contrib.frontend_forms' in settings.INSTALLED_APPS:
-    from djangocms_frontend.contrib.frontend_forms.cms_plugins import (
+    from djangocms_frontend.contrib.content.cms_plugins import (
```

Comment:
```suggestion
    from djangocms_frontend.contrib.frontend_forms.cms_plugins import (
```
This is a compatibility shim that allows users who use an old version of djangocms-frontend which once contained these forms to migrate. I would not change this.